Clarified availability and new navigation's connection (#20919)
cloudflare/cloudflare-docs: src/content/docs/security/web-assets.mdx

This will add the specified endpoints to your list of managed endpoints. You can view your list of managed endpoints in the **Endpoints** tab.

> For save endpoints:

Reviewer: ```suggestion
For saved endpoints:
```

> - Cloudflare will start collecting [performance data](/api-shield/management-and-monitoring/#endpoint-analysis) per endpoint
> - You can use [labeling service](/api-shield/management-and-monitoring/endpoint-labels/) to organize your endpoints by such as use cases

Reviewer: ```suggestion
- Cloudflare will start collecting [performance data](/api-shield/management-and-monitoring/#endpoint-analysis) per endpoint.
- You can use the [labeling service](/api-shield/management-and-monitoring/endpoint-labels/) to organize your endpoints by use case.
```
